src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -41,7 +41,7 @@
     <None Include="$(BuildCommonDirectory)NOTICES.txt" Pack="true" PackagePath="\" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' != '$(NETFXTargetFramework)' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == '$(NETCoreTargetFramework)' ">
     <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="runtime" GeneratePathProperty="true" />
```

Comment:
@rainersigwald is it recommended to reference the MSBuild packages on all frameworks or just on the ones where references aren't available? Asking as in L35-L36, msbuild references are used from the .NET Framework installation when targeting netfx.

Reply:
The .NET Framework references are the "most compatible" option, but . . . do we really support modern Pack-in-MSBuild in Visual Studio 2010? That doesn't seem required.

I would lean toward "reference the packages at the explicit lowest-supported-version for the netfx output".